src/app/pages/home-page/offer-item-page/offer-item-page.component.scss

```diff
@@ -1,18 +1,24 @@
 @import '/src/app/styles/variables.scss';
 @import '/src/app/styles/components/tabs.scss';
 @import '/src/app/styles/components/addresses.scss';
+@import '/src/app/styles/custom-theme.scss';
 
 .button {
   text-transform: uppercase;
-  color: $blue-primary;
 
-  &--review {
+  &_review {
```

Comment:
for example
white color on blue background - button_c_primary;
blue color on transparent background - button_c_default

Reply:
did it